src/views/document/RouteView.vue

```diff
@@ -78,12 +78,13 @@
         <div class="box">
           <markdown-section :document="document" :field="fields.summary" />
           <markdown-section v-if="locale.route_history" :document="document" :field="fields.route_history" />
-            <div v-else-if="activities.includes('snow_ice_mixed') || activities.includes('mountain_climbing') || activities.includes('rock_climbing') || activities.includes('ice_climbing') || activities.includes('via_ferrata') || activities.includes('slacklining') || activities.includes('skitouring') && ['5.1', '5.2', '5.3', '5.4', '5.5','5.6'].includes(doc.ski_rating))" class="missing-history-banner no-print"
-              <edit-link v-if="isEditable" :document="document" :lang="lang">
-               <v-translate/> Missing history/>
-              </edit-link>
-            </div>
-            <div v-else disable </div>
+             <div v-else-if="PromoteHistory" class="missing-history-banner no-print" />
+               <edit-link v-if="isEditable" :document="document" :lang="lang" v-translate/> 
+                  "missing_history"
```

Comment:
Do not use quotes.
Also, the i18n for c2c_ui is using mostly english sentences to esae the translation work. You could .e.g use `History is missing. Edit the route to complete the information.`